uniffi_bindgen/src/interface/mod.rs

```diff
@@ -404,34 +413,32 @@ impl<'ci> ComponentInterface {
     /// The set of FFI functions is derived automatically from the set of higher-level types
     /// along with the builtin FFI helper functions.
     pub fn iter_ffi_function_definitions(&self) -> Vec<FFIFunction> {
-        self.objects
-            .iter()
-            .map(|obj| {
-                vec![obj.ffi_object_free().clone()]
-                    .into_iter()
-                    .chain(obj.constructors.iter().map(|f| f.ffi_func.clone()))
-                    .chain(obj.methods.iter().map(|f| f.ffi_func.clone()))
-            })
-            .flatten()
+        vec![]
```

Comment:
How `ffi_function_definitions` are rendered hasn't change yet, but this is the beginning of that idea.